pkg/openapi/server.spec.yaml

```diff
@@ -256,6 +424,38 @@ components:
                 machine:
                   flavorId: c7568e2d-f9ab-453d-9a3a-51375f78426b
                   replicas: 3
+                firewall:
```

Comment:
Examples are obviously the best illustration to review 😸 

Looks good so far.  I'd have a word with Adam, the firewall rules may need a description to make the UI happier.